client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -158,60 +129,60 @@ const ProfessionalEmailCard: FunctionComponent< EmailProvidersStackedCardProps >
 			return;
 		}
 
-		const { productsList, selectedDomainName, selectedSite, shoppingCartManager } = props;
-
-		const cartItem = titanMailMonthly( {
+		const props: {
+			meta?: string;
+			domain?: string;
+			source?: string;
+			quantity?: number | null;
+			extra?: RequestCartProductExtra & { email_users: any };
```

Comment:
Could we just add `email_users` to `ResponseCartProductExtra`? (And give it a type other than `any`?)

Reply:
That's a very good idea 👍.